docs/section23/Rule23-2.md

```diff
@@ -0,0 +1,50 @@
+
+# Airside System - Rule 23-2  
+
+**Rule ID:** 23-2  
+**Rule Description:** System 5 & 7 minimum volume setpoint shall be 30% of zone peak airflow, minimum outdoor airflow, or rate required to comply with minium accreditation standards whichever is larger.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 23 Air-side  
+**Appendix G Section Reference:** Section G3.1.3.13 VAV Minimum Flow Set Points (Systems 5 and 7)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-5 or 7  
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:**  
+
+1. baseline_hvac_type()  
+
+## Rule Logic:  
+
+- For each zone in B_RMR: `for zone_b in B_RMR...zones:`
+
+  - For each terminal in zone: `for terminal_b in zone_b.terminals:`
+
+    - Get HVAC system serving terminal: `hvac_b = terminal_b.served_by_heating_ventilation_air_conditioning_systems`
+  
+      - Check if HVAC system is type 6 or 8: `if baseline_hvac_type(hvac_b) in ["SYS-5", "SYS-7"]:`
+
```

Comment:
update line 30 to say if HVAC system is 5 or 7

Reply:
Corrected.